### PR TITLE
[Merged by Bors] - feat: Preimage of pointwise multiplication

### PR DESCRIPTION
**Pull request by YaelDillies**
From PFR

Co-authored-by: Patrick Massot <patrickmassot@free.fr>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Pointwise.lean [hunk 1]
```diff
@@ -2309,4 +2310,42 @@ theorem Finite.toFinset_vsub (hs : s.Finite) (ht : t.Finite) (hf := hs.vsub ht)
 
 end VSub
 
+section MulAction
+variable [Group α] [MulAction α β]
+
+@[to_additive]
```
Comment (sgouezel) [inline Mathlib/Data/Finset/Pointwise.lean:2316]: should it be a simp lemma?

File: Mathlib/Data/Finset/Pointwise.lean [hunk 2]
```diff
@@ -2309,4 +2310,42 @@ theorem Finite.toFinset_vsub (hs : s.Finite) (ht : t.Finite) (hf := hs.vsub ht)
...
+    simpa [-Finset.coe_mul] using Finset.card_mul_le
+  all_goals simp
+
+end IsCancelMul
+
+section Group
+variable [Group α] {s t : Set α}
+
+@[to_additive (attr := simp)]
+lemma card_inv (s : Set α) : Nat.card ↥(s⁻¹) = Nat.card s := by
```
Comment (sgouezel) [inline Mathlib/Data/Finset/Pointwise.lean:2343]: InvolutiveInv would be enough for this one.

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

File: Mathlib/Data/Finset/Pointwise.lean (hunk 2 above)
Comment (YaelDillies) [reply Mathlib/Data/Finset/Pointwise.lean:2343]: Well spotted

**Comment by YaelDillies** [comment]
bors merge

**Comment by YaelDillies** [comment]
Whoops, typo fixed

bors merge

**Pull request closed without merging**